Objects/exceptions.c

```diff
@@ -1981,6 +1981,13 @@ SimpleExtendsException(PyExc_Warning, SyntaxWarning,
                        "Base class for warnings about dubious syntax.");
 
 
+/*
+ *    3xWarning extends Warning
+ */
+SimpleExtendsException(PyExc_Warning, Py3xWarning,
```

Comment:
It feels like we probably want to break this exception class out into its own commit if possible (and then we can explain the commit message why we've done it, what the implications are, etc).

Reply:
The associated PR might have been merged but for some reason the diff still shows even after merging wierd. We can see what do with @vext01

Reply:
Did you rebase?

Reply:
I might have just started a new branch off the numbers one, then opened the PR, since it was opened before the numbers PR was merged

Reply:
If @vext01 is OK with you doing so, it might be possible (and worthwhile) to do a rebase now and see if that sorts things out. I'll leave that to the two of you!

Reply:
I'm OK with that.